main.go

```diff
@@ -120,14 +122,31 @@ func readPackages() {
 
 // modified will use git to find out which folders have been changed
 func modified() {
-	cmdArgs := []string{
-		"--no-pager",
-		"-C",
-		*gitDirPtr,
-		"diff",
-		"--name-only",
-		*commitFromPtr,
-		*commitToPtr,
+
+	var cmdArgs []string
+	switch *includeTestFiles {
+	case true:
+		cmdArgs = []string{
+			"--no-pager",
+			"-C",
+			*gitDirPtr,
+			"diff",
+			"--name-only",
+			*commitFromPtr,
+			*commitToPtr,
+		}
+	default:
+		cmdArgs = []string{
+			"--no-pager",
+			"-C",
+			*gitDirPtr,
+			"diff",
+			"--",
+			`":!*test*"`,
```

Comment:
Test files in Go end with a `_test.go` suffix.

There may be files with names that contain **test**.
* con**test**ant
* la**test**
* **test**imonial
* etc.

Reply:
ahhh good thought.. i'll be more explicit